src/status_im/subs/communities.cljs

```diff
@@ -67,6 +61,41 @@
           (sort-by #(get names (get % 0)))
           (sort-by #(visibility-status-utils/visibility-status-order (get % 0)))))))
 
+(re-frame/reg-sub
+ :communities/sorted-community-members-section-list
+ (fn [[_ community-id]]
+   (let [profile                   (re-frame/subscribe [:profile/profile])
+         members                   (re-frame/subscribe [:communities/community-members community-id])
+         visibility-status-updates (re-frame/subscribe [:visibility-status-updates])
+         my-status-update          (re-frame/subscribe [:multiaccount/current-user-visibility-status])]
+     [profile members visibility-status-updates my-status-update]))
+ (fn [[profile members visibility-status-updates my-status-update] _]
+   (let [online? (fn [public-key]
+                   (let [status-update          (if (or (string/blank? (:public-key profile))
+                                                        (= (:public-key profile) public-key))
+                                                  my-status-update
+                                                  (get visibility-status-updates public-key))
+                         visibility-status-type (:status-type status-update)]
+                     (or (= visibility-status-type constants/visibility-status-automatic)
+                         (= visibility-status-type constants/visibility-status-always-online))))
+         names   (reduce (fn [acc contact-identity]
+                           (assoc acc
+                                  contact-identity
+                                  (when (= (:public-key profile) contact-identity)
+                                    (:primary-name profile)
+                                    contact-identity)))
+                         {}
+                         (keys members))]
+     (->> members
+          (sort-by #(get names (get % 0)))
+          keys
+          (group-by (fn [item]
+                      (online? item)))
+          reverse
+          (map (fn [[k v]]
+                 {:title (if k :Online :Offline)
```

Comment:
Any reason to use keywords instead of strings as the titles? Normally keywords in clojure are only lowercase.